docs/learn/bossroom/getting-started-boss-room.md

```diff
@@ -6,200 +6,275 @@ description: Learn more about installing and running the Boss Room game sample.
 
-## Test multiplayer
+### Registering the project with Unity Gaming Services (UGS)
 
-To see the multiplayer functionality in action, you can either run multiple instances of the game [locally on your computer](#local-multiplayer-setup) or choose to [connect through the internet](#multiplayer-over-internet).
+Boss Room leverages several services from UGS to facilitate connectivity between players. To use these services inside your project, you must [create an organization](https://support.unity.com/hc/en-us/articles/208592876-How-do-I-create-a-new-Organization-) inside the Unity Dashboard, and enable the [Relay](https://docs.unity.com/relay/get-started.html) and [Lobby](https://docs.unity.com/lobby/game-lobby-sample.html) services. Otherwise, you can still use Boss Room without UGS.
 
-## Local multiplayer setup
+### Opening the project for the first time
 
-For a local multiplayer setup, you must build an executable and launch several instances of this executable to both host and join a game.
+Once you have downloaded the project, follow the steps below to get up and running:
 
-1. With the Boss Room project open in your Unity editor, click **File** > **Build Settings** > **Build**.
-2. Save the binary as `Boss Room`.
+1. Check that you have installed the most recent [LTS editor version](https://unity.com/releases/2021-lts).
+    1. Include standalone support for **Windows/Mac** in your Unity Editor installation.
+2. Add the project to the Unity Hub by selecting the **Add** button and pointing it to the root folder of the downloaded project.
+    1. Unity imports all the project assets the first time you open the project, which takes longer than usual.
+3. Select the **Play** button. You can then host a new game or join an existing one using the in-game UI.
 
-After the build has completed, you can launch several instances of the built executable to both host and join a game.
+### Exploring the project
 
-:::important Mac Users
-To run multiple instances of the same app, you need to use the command line:
-1.  First, change your directory to the folder where you saved the Boss Room executable. For example, `cd Desktop/com.unity.multiplayer.samples.coop-<version>`.
-2.  Run the command `Open -n YourAppName.app`. If you saved the app as `BossRoom`, your command is `Open -n BossRoom.app`. However, if you saved your app as `Boss Room` with a space, your command needs to include quotation marks ("") around the executable name: `Open -n "BossRoom.app"`.
-:::
+Boss Room is an eight-player co-op RPG game experience, where players collaborate to fight imps, and then a boss. Players can select between classes that each have skills with didactically interesting networking characteristics. Control model is click-to-move, with skills triggered by a mouse button or hotkey.
```

Comment:
Just a suggestion:
Didactically is a less commonly used word and the intent behind this sentence could be lost/passed over.

Perhaps something like the following?

> Players can select between character classes that were designed to provide educational oriented implementations of characteristics commonly found in similar small-scale netcode games.

Reply:
Beautiful 🙏